modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,197 @@
+
+- `monitoring-center-metrics-api` to validate Influx DB queries and proxy the calls to the data source (`dias-prov-k8s-am-influxdb-comp`).
+
+=== Visualizer Components
+
+Visualizer components represents the "Anypoint Visualizer" product.
+
+Logstash `dias-prov-k8s-am-ingestor-comp` sends a subset of mule agent events to the `visualizer-topology-processor` service.
+That service process those events and stores the results in the Postgres DB (`stolon-amv`).
+The final application network generation will be handled by `visualizer-topology-api` service and is invoked by a user traffic.
+There is a set of cron jobs to prevent the uncontrolled DB growth for mule agent events. They perform the stale data clean and runs as part of `visualizer-janitor` service.
+
+All user traffic comes to the `visualizer-experience-api` service that orchestrates loading all necessary data from different data sources (like `visualizer-topology-api`, `monitoring-center-ui-api`, etc.) and performs further data decorations.
+
+== Considerations
+
+=== Cluster secrets
+
+All Anypoint Monitoring/Visualizer components use self-signed certificates for TLS support. Those certificates are generated by the `dias-prov-k8s-amr-certs-comp` components and stored as secrets within the cluster under the `dias` namespace. It is important to notice if the customer changes the platform DNS, the `dias-prov-k8s-amr-certs-comp` will be redeployed and secrets will be updated. Additionally, the `dias-prov-k8s-amr-ingestor-router-comp` component, running Nginx, will be redeployed as well, refreshing those new secrets.
+
+=== Stateful sets directories
+
+Each stateful set uses different directories to hold its data, resulting in the following directories:
+
+```
+  /var/lib/data/influxdb/data
+  /var/lib/data/influxdb/meta
+  /var/lib/data/logstash
+  /var/lib/data/dias-cassandra
+  /var/lib/data/dias-meld
+```
+
+InfluxDB meta pods (3 pods) use `/var/lib/data/influxdb/meta` while data pods (2 pods) use `/var/lib/data/influxdb/data`. Logstash uses `/var/lib/data/logstash` and finally Insight uses the remaining two directories to hold its data.
+
+== Requirements
+
+Hardware requirements are listed in the main PCE hardware section but 3 nodes are required for all Anypoint Monitoring/Visualizer workload.
+
+Port 8895 needs to be accesible in the main balancer along with the rest of the required ports.
+
+
+=== InfluxDB License
+
+InfluxDB requires a valid license to operate, which should have been provided at the time of purchase of Anypoint Monitoring.
+
+Anypoint Monitoring requires a Kubernetes secret to be present in the cluster before all Anypoint Monitoring/Visualizer charts are deployed.
+
+The following command needs to be run in order to create the required secret in the default namespace:
+
+
+```
+kubectl create secret generic influxdb-license --from-literal=license={license}
```

Comment:
This instructions should also be part of the main PCE 3.0 Install Instructions

Reply:
Where's that located? I can copy that over.

Reply:
@alejandrorascovan Should that go into the runbook?

Reply:
@juanpablotimpanaro Let's actually just remove this from the public docs since services is supposed to perform the installation and they already have this step explained in the wiki

Reply:
Done